universe/config.json

```diff
@@ -71,6 +71,11 @@
                     "description": "Launch the Dispatcher using the Universal Container Runtime (UCR)",
                     "default": false
                 },
+                "docker_user": {
```

Comment:
Should we add a note that this is *only* required if it **differs** from `service.user`?

Reply:
I updated description, please check if it reflects the purpose of this property clearly.